projects/core/src/auth/user-auth/http-interceptors/auth.interceptor.ts

```diff
@@ -65,7 +65,8 @@ export class AuthInterceptor implements HttpInterceptor {
                 } else if (
                   // Refresh the expired token
                   // Check if the OAuth endpoint was called and the error is because the refresh token expired
-                  this.errorIsInvalidToken(errResponse)
+                  this.errorIsInvalidToken(errResponse) ||
+                  this.isOidcInvlidToken(errResponse)
```

Comment:
maybe insert logic into errorIsInvalidToken?

Reply:
if someday we decide to keep spartacus for a single branch, then seems split it to a function is more easy to do adaptation :)

Reply:
as you already have judgement as below so i guess it's safe to put them into one routine? BTW. Does it only for OIDC(I just wonder whether it's best name for the logic)? 

 if (!this.authConfigService.getOAuthLibConfig().disablePKCE && authHeader)

Reply:
moved logic to errorIsInvalidToken